components/Repositories/RossChart.tsx

```diff
@@ -0,0 +1,137 @@
+import { FaUsers } from "react-icons/fa6";
+import { Bar, BarChart, CartesianGrid, ResponsiveContainer, Tooltip, TooltipProps, XAxis, YAxis } from "recharts";
+import { NameType, ValueType } from "recharts/types/component/DefaultTooltipContent";
+import { useMemo } from "react";
+import Card from "components/atoms/Card/card";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
+import { humanizeNumber } from "netlify/og-image-utils";
```

Comment:
We have one for this also

```suggestion
import { humanizeNumber } from "lib/utils/humanizeNumber";
```

Reply:
The one in `netlify/og-image-utils` is because edge functions aren't aware of the Next.js codebase, so +1 to @brandonroberts suggestion.